### PR TITLE
Add the WorkOrders.finalize documentation

### PR DESCRIPTION
**Pull request by JordyDeGroeve**
Do this according to the proposal #290

### CODE REVIEW

**Review by tiagobastos — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3650,6 +3650,17 @@ Add a draft work order.
             + type: `workOrder` (string)
             + id: `7c1d8672-f502-4333-9ea4-7a45add15115` (string)
 
+### workOrders.finalize [POST /workOrders.finalize]
+
+ Finalize a draft work order by generating the PDF.
```
Comment (WouterSioen) [inline apiary.apib]: I'm not sure "by generating the PDF" is correct. Something like `which will generate the work order pdf` might make more sense?

**Review by WouterSioen — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (JordyDeGroeve) [reply apiary.apib]: Just copied the description from the proposal. I like`which will generate ..` better, but will wait for some more input on this before changing, if you don't mind.

Comment (dietrichm) [reply apiary.apib]: Let's change this. Generating is a side effect, not the main thing of finalizing. The main thing is just the state of the WO that is being changed.

File: apiary.apib [hunk 2]
```diff
@@ -3650,6 +3650,17 @@ Add a draft work order.
...
+
+ + Request (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+
+ + Response 204
```
Comment (dietrichm) [inline apiary.apib]: B: excess indenting (there's a space in front of every line).

**Review by dietrichm — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 2 above)
Comment (JordyDeGroeve) [reply apiary.apib]: Really? 😓 This should get picked up by the linter..

**Review by tiagobastos — APPROVED** [review]

File: src/09-workorders/work-orders.apib [hunk 3]
```diff
@@ -113,3 +113,14 @@ Update a draft work order.
         + location (Address, optional, nullable)
 
 + Response 204
+
+### workOrders.finalize [POST /workOrders.finalize]
+
+Finalize a draft work order. This will also generate the work order pdf.
```
Comment (dietrichm) [inline src/09-workorders/work-orders.apib]: PP:
```suggestion
Finalize a draft work order. This will also generate the work order PDF.
```

**Review by dietrichm — APPROVED** [review]

**Review by dietrichm — DISMISSED** [review]

**Comment by JordyDeGroeve** [comment]
That one time I used the `commit suggestion` option was the one time I shouldn't.

**Review by WouterSioen — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Comment by dietrichm** [comment]
✅ Checked

**Pull request merged**